Sources/eppo/EppoClient.swift

```diff
@@ -117,6 +151,7 @@ public class EppoClient {
         guard !alreadyLoaded else { return }
         
         try await self.configurationStore.fetchAndStoreConfigurations()
+        self.isOnlineMode = true
```

Comment:
🐛  `loadIfNeeded` is called in an async context and `isOnlineMode` is set only after configuration is successfully fetched and stored, so the user might call assignment functions before `isOnlineMode` is set.

This nullifies the checks in `getInternalAssignment()` because if `isOnlineMode` is `true`, that means that we fetched the configuration successfully, and therefore api key and host are valid.